deploy/gk-deploy

```diff
@@ -489,18 +489,32 @@ if [[ $? -ne 0 ]]; then
 fi
 output "OK"
 
+s=0
 heketi_service=""
 debug -n "Determining heketi service URL ... "
 while [[ "x${heketi_service}" == "x" ]] || [[ "${heketi_service}" == "<none>" ]]; do
-  heketi_service=$(${CLI} describe svc/deploy-heketi | grep "Endpoints:" | awk '{print $2}')
+  if [[ ${s} -ge ${WAIT} ]]; then
+    debug "Timed out waiting for deploy-heketi service."
+    break
+  fi
   sleep 1
+  ((s+=1))
+  if [[ "${CLI}" == *oc\ * ]]; then
+    heketi_service=$(${CLI} describe routes/deploy-heketi | grep "Requested Host:" | awk '{print $3}')
```

Comment:
Technically I did, but I realize that I did not test the failure condition. D'oh. :) Will fix.